packages/wonder-blocks-dropdown/src/components/option-item.tsx

```diff
@@ -301,16 +301,65 @@ export default class OptionItem extends React.Component<OptionProps> {
     }
 }
 
-const {blue, white, offBlack} = color;
-
 const focusedStyle = {
     // Override the default focus state for the cell element, so that it
     // can be added programmatically to the button element.
     borderRadius: spacing.xxxSmall_4,
-    outline: `${spacing.xxxxSmall_2}px solid ${color.blue}`,
+    outline: `${spacing.xxxxSmall_2}px solid ${semanticColor.border.focus}`,
     outlineOffset: -spacing.xxxxSmall_2,
 };
 
+const theme = {
+    root: {
+        default: {
+            background: semanticColor.surface.primary,
+            foreground: semanticColor.text.primary,
+        },
+        hover: {
+            background:
+                semanticColor.action.filled.progressive.hover.background,
+            foreground:
+                semanticColor.action.filled.progressive.hover.foreground,
+        },
+        press: {
+            background:
+                semanticColor.action.filled.progressive.press.background,
+            foreground:
+                semanticColor.action.filled.progressive.press.foreground,
+        },
+        disabled: {
+            background: "transparent",
+            foreground: semanticColor.text.disabled,
+        },
+    },
+    checkbox: {
```

Comment:
Same question about if checkbox styles need to stay in sync with the Checkbox component from the form package!